CONTRIBUTORS.md

```diff
@@ -22,3 +22,4 @@
 |18| [Li Jin](https://github.com/sljtheultima) | |Malaysia|
 |19| [Peter Irvine](https://github.com/peirvine) | UMN | USA |
 |20| [Eric Moynihan](https://github.com/TooFiveFive) | UCC | Ireland |
+|21| [James Flynn](https://github.com/james-flynn-ie) | | |
```

Comment:
If you have no problem, please consider adding your institution name and country, else let me know and I will merge without them.